test/__mocks__/my-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "verbose": false,
```

Comment:
Could you change this to `true`? We'd be able to see the config in the snapshot then.

Reply:
Sure